RecoLocalMuon/RPCRecHit/src/DTSegtoRPC.cc

```diff
@@ -371,12 +365,12 @@ DTSegtoRPC::DTSegtoRPC(edm::Handle<DTRecSegment4DCollection> all4DSegments, cons
 		    }
 		   
 		    if(debug)  std::cout<<"Calling to Object Map class"<<std::endl;
-		    ObjectMap* TheObject = ObjectMap::GetInstance(iSetup);
+		    ObjectMap* TheObject = new ObjectMap(iSetup);
```

Comment:
Better would be to do
```
auto TheObject = std::make_unique<ObjectMap>(iSetup);
```
This guarantees that even if an exception is thrown the object will be properly deleted.